pkg/repository/notificationrepository/notificationRepository.go

```diff
@@ -30,16 +31,15 @@ func NewNotificationRepository(db *sqlx.DB) (port.NotificationRepository, error)
 }
 
 func (r *NotificationRepository) ListNotifications(ctx context.Context, resultSelector query.ResultSelector) (notifications []models.Notification, totalResults uint64, err error) {
+	log.Debug().Msgf("list notifications")
```

Comment:
Is this statement intended to be there ? It does not need `Msgf` as there are is no format string.

Reply:
Remnants from debugging. It has been removed